xla/service/latency_hiding_scheduler.h

```diff
@@ -1164,6 +1164,8 @@ class LatencyHidingScheduler : public HloModulePass {
   absl::flat_hash_set<HloComputation*> computations_to_schedule_;
 };
 
+static constexpr absl::string_view kPeakMemoryUsage = "peak_memory_usage";
```

Comment:
This pollutes the entire xla namespace. Since it's just for verification & debugging I would either move it somewhere else (within LHS) or keep raw str approach.